### PR TITLE
Re-Numbered and added the last few

### PR DESCRIPTION
**Pull request by adam-ma-roberts**
Re-Numbered after some removals and added the last few, now up to 23.

### CODE REVIEW

File: updater/nd_commander_tips/translations/nd_commander_tips.phrases.txt [hunk 1]
```diff
@@ -115,19 +115,38 @@
...
+		"en"		"Bots respond faster to the Attack command. Place on resource points so they capture"
 	}
 	
-	"Title"
+	"Radar for stealths"
 	{
-		// Note
-		"en"		"Tip"
+		// 20 - Remind Commander that placing a radar near areas plagued with enemy stealths can help
+		"en"		"Near areas that are being attacked by stealths, Radar installations can help your troops"
```
Comment (stickz) [inline updater/nd_commander_tips/translations/nd_commander_tips.phrases.txt]: Radar Stations.

File: updater/nd_commander_tips/translations/nd_commander_tips.phrases.txt [hunk 2]
```diff
@@ -115,19 +115,38 @@
...
+	{
+		// 22 - Remind them to keep upgrading Commander Damage, (and hopefully the other research) as, as the game goes on it needs to get stronger
+		"en"		"As the game progresses you need to keep Commander Damage and other research, up to date, so as to keep it an effective tool"
+	}
+	
+	"Lost Prime"
+	{
+		// 23 - Remind commander to tag prime for capture after it is lost
+		// Could this be triggered with the other lost prime alert ? 
+		"en"		"Consider Tagging Prime for re-capture"
```
Comment (stickz) [inline updater/nd_commander_tips/translations/nd_commander_tips.phrases.txt]: Consider tagging prime with the capture order, to notify your team.

File: updater/nd_commander_tips/translations/nd_commander_tips.phrases.txt [hunk 3]
```diff
@@ -115,19 +115,38 @@
...
+	"Commander Damage"
+	{
+		//  21 - Remind them that commander damage is expensive (for those who over use) but that it is useful (for those who do not)
+		"en"		"Commander Damage, though expensive is very useful somtimes"
+	}
+	
+	"Upgrade Commander Damage"
+	{
+		// 22 - Remind them to keep upgrading Commander Damage, (and hopefully the other research) as, as the game goes on it needs to get stronger
+		"en"		"As the game progresses you need to keep Commander Damage and other research, up to date, so as to keep it an effective tool"
```
Comment (stickz) [inline updater/nd_commander_tips/translations/nd_commander_tips.phrases.txt]: As the game progresses, research commander abilities 2 to unlock walls.

File: updater/nd_commander_tips/translations/nd_commander_tips.phrases.txt (hunk 2 above)
Comment (adam-ma-roberts) [reply updater/nd_commander_tips/translations/nd_commander_tips.phrases.txt]: Done

File: updater/nd_commander_tips/translations/nd_commander_tips.phrases.txt (hunk 3 above)
Comment (adam-ma-roberts) [reply updater/nd_commander_tips/translations/nd_commander_tips.phrases.txt]: As the game progresses, research Commander Abilities II to unlock Walls and increase Commander Damage strength

**Pull request merged**